src/OrientationAPI/WebApplication1/DataAccess/CustomerDataAccess.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using Dapper;
+using WebApplication1.Models;
+
+
+namespace WebApplication1.DataAccess
+{
+     public class CustomerDataAccess : IRepository<CustomerListResult>
+		{
+
+        public List<CustomerListResult> GetAll()
+        {
+            using (var connection =
+                   new SqlConnection(ConfigurationManager.ConnectionStrings["Bangazon"].ConnectionString))
+
+            {
+
+                connection.Open();
+
+                var result = connection.Query<CustomerListResult>
+                                              ("select * from Customer");
+
+                return result.ToList();
+            }
+        }
+
+        public bool InactivateCustomer(int entitytoUpdate)
```

Comment:
Your team should come to a consensus on formatting tabs v. spaces(how many).

Reply:
We discussed this last night and decided to use tabbing.  This will be addressed in our next push.

Reply:
OK, sounds good.

Reply:
We fixed this formatting